e2e/client/playwright/correct-item-corrections-workflow.spec.ts

```diff
@@ -0,0 +1,53 @@
+import {test, expect} from '@playwright/test';
+import {Authoring} from './page-object-models/authoring';
+import {Monitoring} from './page-object-models/monitoring';
+import {restoreDatabaseSnapshot, s, sleep} from './utils';
+import {getStorageState} from './utils/storage-state';
+
+test.use({
+    storageState: getStorageState({corrections_workflow: true}),
+});
+
+test('can correct published item', async ({page}) => {
+    const monitoring = new Monitoring(page);
+    const authoring = new Authoring(page);
+
+    await restoreDatabaseSnapshot();
+    await page.goto('/#/workspace/monitoring');
+
+    await expect(
```

Comment:
This is unreliable because if it was loading slowly, it would pass as "not visible" before finishing loading and becoming visible.

Before using `not.toBeVisible` ensure the view has finished loading.